packages/blocks/src/api/serializer.js

```diff
@@ -120,6 +123,9 @@ export function getSaveElement(
 	innerBlocks = []
 ) {
 	const blockType = normalizeBlockType( blockTypeOrName );
+
+	if ( ! blockType?.save ) return null;
```

Comment:
What are the changes in this function about?

Reply:
Added an inline comment to the changes further above